modules/node/cms_node_api.e

```diff
@@ -212,6 +212,14 @@ feature -- Access: Node
 			Result := node_storage.nodes
 		end
 
+
+	trash_nodes (a_user: CMS_USER): LIST [CMS_NODE]
```

Comment:
please rename as "trashed_nodes" 
I understand this means  "nodes of the trash"
but it looks too close to trash_node (a_node)